script/jetstream-dryrun.js

```diff
@@ -0,0 +1,19 @@
+// Cloud Function performing dry runs.
```

Comment:
Can you add a note here that this is a non-authoritative reference copy of the cloud function called by jetstream.dryrun.dry_run_query with a link to the GCP console? I don't anyone to expect changes here to auto-deploy :D